src/ui/mainwindow.h

```diff
@@ -122,6 +123,8 @@ private slots:
     QMenu *m_backMenu = nullptr;
     QMenu *m_forwardMenu = nullptr;
 
+    QTimer *m_deferOpenUrl;
```

Comment:
It would be nice to have a more sensible name here, at least like `m_openDocsetTimer` or so. Also please add `= nullptr`.